samples/errors7.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+)
+
+type BadRequestError string
+
+var ErrAbortHandler = errors.New("net/http: abort handler")
+
+type Error interface {
```

Comment:
What is the purpose of this interface? It doesn't seem to be used anywhere, and as mentioned before, does not seem to be the right way to do custom error types (DNSError below is)

Reply:
That's right, I've removed it